export const ERROR_OCCURED_DURING_LOOP = "An error occurred during test"
